lib/beaker/test_suite_result.rb

```diff
@@ -150,6 +150,23 @@ def print_test_result(test_case)
       "  Test Case #{test_case.path} #{test_reported}"
     end
 
+    # Saves failure and error cases as an options file for only-failures processing
+    #
+    # @param [Boolean] only_failures    Are we rerunning with fails/errors?
+    # @param [String] save_results_file Where to put the results if so
+    #
+    def persist_results(only_failures, save_results_file)
```

Comment:
`save_results_file` implies a boolean to me -- I think `filepath` or something like that would make more sense as the name of this argument. Also, maybe it makes sense to reverse the order of the arguments and default `only_failures` to false, in case this is useful for other purposes?

The junit file method is called `write_junit_xml` -- maybe it's worth following that naming format here, too, like `write_results_json` -- but I'm not sure how helpful that really is.

Reply:
I changed the method name to persist_test_results to be more consistent with companion methods around test results, and changed the parameter to filepath.